aries_cloudagent/protocols/connections/v1_0/routes.py

```diff
@@ -318,6 +339,10 @@ async def connections_create_invitation(request: web.BaseRequest):
     alias = request.query.get("alias")
     public = json.loads(request.query.get("public", "false"))
     multi_use = json.loads(request.query.get("multi_use", "false"))
+    body = await request.json()
```

Comment:
It doesn't look like the method currently requires a JSON message body, so will this blow up if current controllers aren't adjusted? It could be modified to only parse the request body as JSON if one is provided.

Reply:
It's a POST: it has a body, necessarily

Reply:
Ha ha! It turns out you were right: POST doesn't need a body. All these years I never knew! I will fix it today.